metricbeat/etc/beat.yml

```diff
@@ -1,85 +1,87 @@
-metricbeat.modules:
+###################### Metricbeat Configuration Example #######################
 
-  # Apache Module
-  - module: apache
-    metricsets: ["status"]
-    enabled: true
-    period: 1s
+#==========================  Modules configuration ============================
+metricbeat.modules:
 
-    # Apache hosts
-    hosts: ["http://127.0.0.1/"]
+#----------------------------- Apache Module ----------------------------------
```

Comment:
Ha ha, of course. I added the headers in the `_beat/confg.yml` files and removed the 2 spaces extra indent from the script (not actually needed). Please have a look at the results.